packages/react-composites/tests/browser/chat/fake-adapter/ErrorBar.test.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { stubMessageTimestamps, waitForChatCompositeToLoad, dataUiId, pageClick } from '../../common/utils';
+import { buildUrlForChatAppUsingFakeAdapter, DEFAULT_FAKE_CHAT_ADAPTER_ARGS, test } from './fixture';
+import { expect } from '@playwright/test';
+import { sendMessage, waitForSendMessageFailure } from '../../common/chatTestHelpers';
+
+const TEST_MESSAGE = 'No, sir, this will not do.';
+
+test.describe('ErrorBar is shown correctly', async () => {
+  test('with wrong thread ID', async ({ page, serverUrl }) => {
+    await page.goto(
+      buildUrlForChatAppUsingFakeAdapter(serverUrl, {
+        ...DEFAULT_FAKE_CHAT_ADAPTER_ARGS,
+        chatThreadClientFunctionErrors: {
+          getProperties: { message: 'Could not get properties', statusCode: 400 },
+          listMessages: { message: 'Could not list messages', statusCode: 400 },
+          sendMessage: { message: 'Could not send message', statusCode: 400 }
+        }
+      })
+    );
+    await waitForChatCompositeToLoad(page);
+    await stubMessageTimestamps(page);
+    expect(await page.screenshot()).toMatchSnapshot('error-bar-wrong-thread-id.png');
+
+    await sendMessage(page, TEST_MESSAGE);
+    await waitForSendMessageFailure(page);
+    await stubMessageTimestamps(page);
+    expect(await page.screenshot()).toMatchSnapshot('error-bar-send-message-with-wrong-thread-id.png');
+    // test resend button in contextual menu
```

Comment:
Tangent: You didn't add this check in this PR, so I agree you should carry it over.

But we should split off tests for resend button from tests for showing the correct error bar. Especially with the speed up we'll get with the fake services, we can push to make our tests be more specific.